### PR TITLE
Populate samples when selecting a new Dart / Flutter / HTML project

### LINKED ISSUE #1326: New playground needs starting code for Flutter and HTML
The new playground starts new pads off with the same code each time, regardless of which layout they're using.

We need a starting point for Flutter code that presents as small an app as possible and a starting point for Dart+HTML code that all three files with some kind of content in them.

### PR DESCRIPTION
**Pull request by johnpryan**
This adds three new routes, `/flutter`, `/dart`, and `/html`. Each route will populate the editor with sample code if there's no backing gist. When the user selects a type from the new-pad dialog, the app re-routes and the sample is shown.

As far as I can tell, routing only works when the app is deployed and new_playground.html replaces index.html, so this won't be fully tested until #1351 is merged

closes #1326

### CODE REVIEW

File: lib/new_playground.dart [hunk 1]
```diff
@@ -548,45 +562,59 @@ class Playground implements GistContainer, GistController {
     }
   }
 
+  Future showNewFlutter() async {
```
Comment (RedBrogdon) [inline lib/new_playground.dart]: Two questions on these three functions:

* Looks like they're all using Layout.dart as the final layout. Is that intentional?
* Could they be collapsed into a single function that took the layout as a parameter and
  - created the sample gist based on that
  - Passed that layout on to _finalizeRoute?

File: lib/src/sample.dart [hunk 2]
```diff
@@ -11,3 +11,70 @@ void main() {
...
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+}
+
+h1 {
+  color: white;
```
[the remark this answers is not in the document]
Comment (johnpryan) [reply lib/src/sample.dart]: It only works for the dark theme currently

File: lib/new_playground.dart (hunk 1 above)
Comment (johnpryan) [reply lib/new_playground.dart]: That's a good point, I'll make that change

**Comment by johnpryan** [comment]
This is ready for another look

File: lib/new_playground.dart [hunk 3]
```diff
@@ -562,44 +562,39 @@ class Playground implements GistContainer, GistController {
...
   }
 
-  Future showNewDart() async {
-    var loadResult = _loadGist();
-    var shouldAutoRun = loadResult == LoadGistResult.storage;
-
-    // If no gist was loaded, use a new Dart gist.
-    if (loadResult == LoadGistResult.none) {
-      editableGist.setBackingGist(createSampleDartGist());
+  Gist _createGist(Layout layout) {
```
Comment (RedBrogdon) [inline lib/new_playground.dart]: nit: "createSampleGist" or "createDefaultGist" would be slightly more descriptive.

**Pull request merged**